pkg/collector/corechecks/sbom/processor_test.go

```diff
@@ -606,8 +608,13 @@ func TestProcessEvents(t *testing.T) {
 				"sbom.container_image.enabled": true,
 			},
 		}),
+		fx.Replace(sysprobeconfigimpl.MockParams{
+			Overrides: map[string]interface{}{
+				"runtime_security_config.sbom.enabled": true,
+			},
+		}),
```

Comment:
Could we simply do `systemProbe.SetWithoutSource("runtime_security_config.sbom.enabled", true)` ?

Reply:
We could :)